pkg/prompts/prompts.go

```diff
@@ -1063,11 +1109,6 @@ func PromptAddress(
 			if err != nil {
 				return "", err
 			}
-		case XChainFormat:
```

Comment:
why removing X-Chain?

Reply:
my bad. not sure how it happened. checking and fixing it back